### ISSUE TITLE
Example: Integration With MySQL 

### ISSUE THREAD

**Issue by WebSpectrum** [riktar/jkanban#138]
Any working examples of updating MySQL column on drag drop?

**Comment by Rampesna** [comment]
Can you explain a little more what you want to do?

**Comment by WebSpectrum** [comment]
yes, I think I need to update  value in MySQL status on dragendEl ... Was wondering if any examples?

**Comment by Rampesna** [comment]
> yes, I think I need to update value in MySQL status on dragendEl ... Was wondering if any examples?

You can look my example project Jotrac, maybe it can help to you.

Script code is here: https://github.com/Rampesna/jotrac/blob/main/resources/views/board/components/script.blade.php
